site/docs/hosting/deno-deploy.md

```diff
@@ -15,27 +15,31 @@ It's just like the other many serverless platforms, but dedicated for Deno apps.
 
 ```ts
 import { webhookCallback } from "https://deno.land/x/grammy/mod.ts";
+import { serve } from "https://deno.land/x/sift@0.5.0/mod.ts";
```

Comment:
Should we specify a version always or never? Or maybe never for grammY, but for everything else?

Reply:
I think the ideal scenario would be to always automatically specify the latest version. But it might be tricky to implement

Reply:
@Loskir The grammY version won't be a problem if not specified, because they are in their own documentation. What do you think about the 3rd parties?

Reply:
It might also not be that tricky to implement the auto-upgrade of the grammY imports. It is just a custom plugin away.

Reply:
Yes, doing automatic replacements in the code is hard, it would either have to be fetched dynamically for each page visit, or we would have to change the docs for every release. If there was VuePress for Deno, we could render things easily on the edge, but this does not exist yet.

I think it's not a problem with our own URLs. All code examples will work, regardless of the version. Also, the Deno LSP automatically suggests to replace the URL with the redirect, so if the code was to be copied into an editor, the latest version can be inserted automatically.

I'm rather concerned about third-party code, as we have no control over it.

Reply:
@KnorpelSenf For 3rd-party modules, I think we can specify the version that worked for us at the time of the writing of the documentation, and we can accept working PQs that upgrade their versions. What do you say?

Reply:
That sounds good, then lets stick with:
1. Never versions for grammY so that people always automatically pull in the lastest code
2. Always versions for third-party modules so that the code examples are guaranteed to work

@Loskir objections?

Reply:
> Also, the Deno LSP automatically suggests to replace the URL with the redirect, so if the code was to be copied into an editor, the latest version can be inserted automatically.

That's why i suggested specifying it in the docs. I'd like the examples to be completely copy-pasteable.

As for third-party libraries: I agree.

Reply:
I think the easiest way to achieve this is by creating a new plugin in the `config.ts` file. I suggest leaving this PQ as-is, and taking care of that on a different branch.

The Deno manual does it via env var syntax: https://github.com/denoland/manual/blob/main/introduction.md#comparison-to-nodejs

We could simply use regular expressions. I am sure that something like `@GRAMMY_VERSION` does not occur anywhere else.

I opened https://github.com/grammyjs/website/issues/341.